sklearn/calibration.py

```diff
@@ -41,9 +41,10 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
 
     Parameters
     ----------
-    base_estimator : instance BaseEstimator
+    base_estimator : BaseEstimator instance, default=None
         The classifier whose output need to be calibrated to provide more
-        accurate `predict_proba` outputs.
+        accurate `predict_proba` outputs. If None, then
+        :class:`~sklearn.svm.LinearSVC` is calibrated.
```

Comment:
Nit:

```suggestion
        accurate `predict_proba` outputs. The default classifier is
        a :class:`~sklearn.svm.LinearSVC`.
```

This is the match the docstring of `StackingRegressor` with `final_estimator`:

```py
    final_estimator : estimator, default=None
        A regressor which will be used to combine the base estimators.
        The default regressor is a `RidgeCV`.
```

Reply:
@thomasjpfan Consistency matters 👍.